drivers/ethernet/Kconfig.mcux

```diff
@@ -46,7 +46,7 @@ config ETH_MCUX_TX_BUFFERS
 	int "Number of MCUX TX buffers"
 	depends on ETH_MCUX
 	default 1
-	range 1 16
+	range 1 1
```

Comment:
Is this change really required in this PR? Or is it along the lines of "don't promise false things"?